### DISCUSSION TITLE
Select component: how to prevent onSearchChange triggering from onChange?

### DISCUSSION THREAD

**Discussion by RedFour** [mantinedev/mantine#3358] [Q&A]
I'm trying to make a searchable select component, where the search input not matching select data can directly be used for mutations. I want user typing in the field to be triggering a different handler than selecting from a dropdown item.

However, when selecting a dropdown item, the onSearchChange is also getting triggered several times. Making it impossible to separate out the two interactions. Is there another "on<...>Change" I can use?

**Comment by rtivital** [comment]
No, it is not supported. There is no way to separate those handlers
